parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveType.java

```diff
@@ -18,16 +18,19 @@
  */
 package org.apache.parquet.schema;
 
-import java.util.Arrays;
```

Comment:
Nit: Please don't move imports around. (Again, harder to pick commits, rebase, etc.)